exporter/otlpexporter/otlp_test.go

```diff
@@ -74,7 +74,7 @@ func otlpReceiverOnGRPCServer(ln net.Listener) *mockReceiver {
 
 func TestSendTraceData(t *testing.T) {
 	// Start an OTLP-compatible receiver.
-	ln, err := net.Listen("tcp", "localhost:")
+	ln, err := net.Listen("tcp", "0.0.0.0:")
```

Comment:
This makes it painful to run the tests on Mac due to security policies. I've been trying to change things to default to localhost (at least for dev builds and test stuff). Can we keep tests to localhost even if the default is 0.0.0.0 when running standalone?

Reply:
That was my intention, but then tests did not pass -- I will spend some more time and figure it out.